FreeRTOS/FreeRTOS-Kernel
# use space to replace tab and remove meaningless space in the end of each line

use space to replace tab and remove meaningless space in the end of each line

Description
-----------
1. use Space(4 spaces) to replace Tab to prevent that there are Spaces and Tab mixed in files.
2. remove meaningless space in the end of each line

Test Steps
-----------


Related Issue
-----------
negative 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
